apps/mobile/src/components/button.tsx

```diff
@@ -66,7 +66,7 @@ export function getTextColor(buttonState: ButtonState) {
   );
 }
 
-// TODO: Move to UI library
+// TODO 289: Move to UI library
```

Comment:
I added this tag as we aren't actually using the `button` from the ui package yet https://github.com/leather-io/issues/issues/289